### PR TITLE
kontribusi DASI

### PR DESCRIPTION
**Pull request by Azhura-DH**
add "saldo & topup" in sidebar

### CODE REVIEW

**Comment by tioirawan** [comment]
I'll check it first, if nothing wrong I'll merge your PR

**Comment by tioirawan** [comment]
just my thought, the top up button looks too "striking"

File: component/siswa/sidebaropen.php [hunk 1]
```diff
@@ -25,6 +25,22 @@
...
+            <div class="sidebar-header">
+                <div class="list">
+                    <ul>
+                        <li class="money">
+                            <div class="bgmoney">
+                                <span><?=rupiah($data["saldo"])?></span>
+                            </div>
+                        </li>
+                        <li class="topup">
+                            <div class="bgtopup">
```
Comment (tioirawan) [inline component/siswa/sidebaropen.php:37]: I think, lets just stick with bootstrap default button

File: styles/sidebar.css [hunk 2]
```diff
@@ -463,4 +463,53 @@ body {
...
+}
+
+.bgmoney {
+    padding: 5px;
+}
+
+.bgtopup {
+    cursor: pointer;
+    padding: 5px 10px;
+    background: linear-gradient(270deg, gold, white, gold);
```
Comment (tioirawan) [inline styles/sidebar.css:526]: I don't think it will fit with the current design

**Review by tioirawan — CHANGES_REQUESTED** [review]

**Comment by indmind** [comment]
latihan cek kebiasaan, aku iko pas google codein yo ditolak terus 😢

**Comment by tioirawan** [comment]
nice

File: component/siswa/sidebaropen.php [hunk 3]
```diff
@@ -25,22 +25,22 @@
...
                 </div>
             </div>
-            <!--<div class="sidebar-header">
+            <div class="sidebar-header">
                 <div class="list">
                     <ul>
                         <li class="money">
                             <div class="bgmoney">
-                                <span>--><?#=rupiah($data["saldo"])?><!--</span>
+                                <span><?=rupiah($data["saldo"])?></span>
```
Comment (tioirawan) [inline component/siswa/sidebaropen.php]: gk keliatan

**Review by tioirawan — CHANGES_REQUESTED** [review]

**Comment by tioirawan** [comment]
everything looks good!

**Pull request merged**